test/FunctionalTests/Tests/DataServices/UnitTests/ServerUnitTests/Tests/RequestUriProcessorTest.cs

```diff
@@ -370,6 +370,9 @@ public void RequestUriProcessorKeySpecialCharsTest()
                 new Dimension("WebServerLocation", new WebServerLocation[]{
                         WebServerLocation.InProcess,
                         WebServerLocation.InProcessWcf
+

```

Comment:
I just added a comment to the file edited @ 72cc649 "remove IIS Local server test cases due to timing issue"